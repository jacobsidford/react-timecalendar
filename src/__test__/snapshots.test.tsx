import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Week from '../Weeks';
import Day from '../Day';
import Header from '../Header';

it('Header should render correctly for snapshot', () => {
  const mockDate = new Date('2020-10-05T12:00:00.000Z').toString(); 
    const mockHeader = shallow(
      <Header
        selectedDate={mockDate}
        nextTime={jest.fn()}
        prevTime={jest.fn()}
      />
      );
	expect(mockHeader).toMatchSnapshot();
});

it('Weeks should render correctly for snapshot', () => {
	const mockDate = new Date('2020-10-05T12:00:00.000Z');
	const mockWeek = shallow(
		<Week
			selectedDate={mockDate}
			onDateClick={() => true}
			selectedTime={{ start: mockDate.toString(), end: mockDate.toString() }}
		/>
	);
	expect(mockWeek).toMatchSnapshot();
});
