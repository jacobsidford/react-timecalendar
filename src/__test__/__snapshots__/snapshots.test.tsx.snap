// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly for snapshot 1`] = `ShallowWrapper {}`;

exports[`Weeks should render correctly for snapshot 1`] = `ShallowWrapper {}`;

exports[`Weeks should render correctly for snapshot 2`] = `ShallowWrapper {}`;

exports[`should render correctly 1`] = `ShallowWrapper {}`;
